src/cloud/release-notes/ece-release-notes.md

```diff
@@ -17,16 +17,98 @@ The `{{site.data.var.ct}}` package uses the following release versioning sequenc
+
+   -  {:.new}**Logging improvements**–Improved log tracking capability by assigning exit codes to critical deploy errors and exposing the exit codes in error message notifications and log events. See [Error message reference for ece-tools]({{site.baseurl}}/cloud/reference/error-codes.html).<!-- MCLOUD-5637, 5531-->
+
+   -  {:.new}**Improved Javascript bundling capabilities**–Now you can use the [Magento Baler module](https://github.com/magento/baler) to build optimized JavaScript and HTML content during static content deployment. See [Optimize JavaScript and HTML content]({{site.baseurl}}/cloud/deploy/static-content-deployment.html#optimize-javascript-and-html-content).<!--MCLOUD-3456-->
+
+   -  {:.new}Improved the process for database dumps (`vendor/bin/ece-tools db-dump`) and updated log messages to clarify that the database dump operation switches the application to maintenance mode, stops consumer queue processes, and disables cron jobs before the dump begins.<!--MCLOUD-5324, MCLOUD-2062-->
+
+   -  {:.fix}Fixed an issue to ensure that the project URL is updated correctly when deploying to Staging and Production environments. Now, `{{site.data.var.ct}}` uses the URL for the route with the `primary:true` attribute set in the project route configuration. See [Deploy variables]({{site.baseurl}}/cloud/env/variables-deploy.html#update_urls).<!--MCLOUD-5883-->
+
+   -  {:.fix}Updated the `generate.xml` build scenario workflow for applying patches. Patches must be applied earlier to update Magento to fix any issues that might cause the `di:compile` and `module:refresh` steps to fail.<!--MCLOUD-5941-->
+
+   -  {:.fix}Fixed an issue in the Magento installation process that incorrectly returns the `Crypt key missing` error. The `crypt/key` value is generated automatically during the installation.<!--MCLOUD-6120-->
+
+-  {:.new}**Service updates**–
+
+   -  {:.new}Added support for PHP 7.4 and MariaDB 10.4.<!--MAGECLOUD-2957, MCLOUD-4144-->
+
+-  {:.new}**Environment variable updates**–
+
+   -  {:.new}Added the **SCD_USE_BALER** variable to enable the Magento Baler module for JavaScript bundling during the {{site.data.var.ece }} build process. See the variable description in the [build variables]({{site.variable}}/cloud/env/variables-build.html#scd_use_baler).<!-- MCLOUD-3456, MCLOUD-3457-->
+
+   -  {:.new}Added the **REDIS_BACKEND** environment variable to configure the Redis backend model for Redis cache for Magento 2.3.5 or later. See the variable description in the [deploy variables]({{site.baseurl}}/cloud/env/variables-deploy.html#redis_backend).<!--MCLOUD-5721, MCLOUD-5865-->
+
+-  {:.new}**CLI command updates**–
+
+   -  {:.new}Added the `wizard:split-db-state` smart wizard to check the status and configuration of the Magento split database performance solution on {{site.data.var.ece}} Pro and Staging environments. See [Smart wizards]({{site.baseurl}}/cloud/deploy/smart-wizards.html).<!--MCLOUD-3734-->
```

Comment:
Is it still okay to keep this here if it is being released later? Or you mean the documentation is released later?